modulefiles/gfsutils_noaacloud.intel.lua

```diff
@@ -2,15 +2,22 @@ help([[
 Build environment for GFS utilities on NOAA Cloud
 ]])
 
-prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.6.0/envs/unified-env/install/modulefiles/Core")
-prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.6.0/envs/gsi-addon-env/install/modulefiles/Core")
-
-local stack_intel_ver=os.getenv("stack_intel_ver") or "2021.3.0"
-local stack_impi_ver=os.getenv("stack_impi_ver") or "2021.3.0"
-local cmake_ver=os.getenv("cmake_ver") or "3.20.1"
+prepend_path("MODULEPATH", "/contrib/spack-stack-rocky8/spack-stack-1.6.0/envs/gsi-addon-env/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/apps/modules/modulefiles")
+prepend_path("PATH", "/contrib/EPIC/bin")
+load("gnu")
+load("stack-intel")
+load("stack-intel-oneapi-mpi")
 
+stack_intel_ver=os.getenv("stack_intel_ver") or "2021.10.0"
 load(pathJoin("stack-intel", stack_intel_ver))
+
+stack_impi_ver=os.getenv("stack_impi_ver") or "2021.10.0"
 load(pathJoin("stack-intel-oneapi-mpi", stack_impi_ver))
+unload("gnu")
```

Comment:
Not sure what is going on here.
loading gnu, intel and impi, then loading a specific version of the intel and impi, and finally unloading gnu?

also, why is PATH being updated here?

Reply:
what is the error without loading GNU?